web/libs/ui/src/tailwind.config.js

```diff
@@ -0,0 +1,190 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./apps/**/*.{js,jsx,ts,tsx}",
+    "./libs/ui/src/**/*.{js,jsx,ts,tsx}",
+    "./libs/storybook/**/*.{js,jsx,ts,tsx}",
+  ],
+  theme: {
+    extend: {
+      colors: {
+        current: "currentColor",
+        transparent: "transparent",
+        stroke: "#EEEEEE",
+        strokedark: "#2D2F40",
+        hoverdark: "#252A42",
+        titlebg: "#ADFFF8",
+        titlebg2: "#FFEAC2",
+        titlebgdark: "#24598F",
+        btndark: "#292E45",
+        white: "#FFFFFF",
+        black: "#181C31",
+        blackho: "#2C3149",
+        blacksection: "#1C2136",
+        primaryho: "#0063EC",
+        meta: "#20C5A8",
+        waterloo: "#757693",
+        manatee: "#999AA1",
+        alabaster: "#FBFBFB",
+        zumthor: "#EDF5FF",
+        socialicon: "#D1D8E0",
```

Comment:
Were you able to discuss the token values with @ricardoantoniocm?

Reply:
yup, we had an agreement on naming, though I would keep these unless we have our tokens in place